source/quickstart.rst

```diff
@@ -0,0 +1,477 @@
+
+Quickstart
+**********
+
+The present tutorial is meant to provide a brief introduction to the use of PySCF to run a multitude of quantum chemical simulations. Starting with input parsing and uncorrelated Hartree-Fock theory for chemical Hamiltonians, we'll incrementally touch upon how to use the majority of methods and features offered by PySCF through a number of simple examples, which all make reference to specific use cases within the dedicated `examples <https://github.com/pyscf/pyscf/tree/master/examples>`_ directory. For brevity and so as to not repeat a number of function calls, please also note that the cells below often share objects in-between one another.
+
+.. _INPUT:
+
+Input Parsing
+=============
+
+Molecules (or unit cells, cf. the :ref:`periodic <PBC>` section) are typically loaded in any of two ways (`gto/00-input_mole.py <https://github.com/pyscf/pyscf/blob/master/examples/gto/00-input_mole.py>`_):
+
+  >>> from pyscf import gto
+  >>> mol_h2o = gto.Mole()
+  >>> mol_h2o.atom = '''O 0 0 0; H 0 1 0; H 0 0 1'''
+  >>> mol_h2o.basis = 'ccpvdz'
+  >>> mol_h2o.build()
+
+or - using the convenient shortcut function - as  
+
+  >>> mol_h2o = gto.M(atom = 'O 0 0 0; H 0 1 0; H 0 0 1', basis = 'ccpvdz')
+
+Calling ``build()`` initializes a bunch of internal control parameters. Whenever you change the value of the attributes of :class:`Mole`, you need to call this function again to refresh the internal data of the object.
+
+Symmetry may be specified in the ``Mole.symmetry`` attribute as either ``True`` or ``False`` (default is ``False``, i.e., off). Alternatively, a particular subgroup can be specified by a string argument (`gto/13-symmetry.py <https://github.com/pyscf/pyscf/blob/master/examples/gto/13-symmetry.py>`_):
+
+  >>> mol_c2 = gto.M(atom = 'C 0 0 .625; O 0 0 -.625', symmetry = 'd2h')
+  
+Numerous other ways of inputting a molecular or crystalline geometry also exist (e.g., by means of Z-matrices or reading in .xyz files), cf. the complete suite of `gto <https://github.com/pyscf/pyscf/blob/master/examples/gto>`_ and `pbc <https://github.com/pyscf/pyscf/blob/master/examples/pbc>`_ examples.
```

Comment:
I think this overlaps too much with the user guide: too much explanation!

Reply:
I will attempt to shorten this.